Gruntfile.js

```diff
@@ -491,13 +491,22 @@ module.exports = function(grunt) {
           ` && curl -X PUT --data '"4294967296"' ${couchConfig.withPath('_node/' + COUCH_NODE_NAME + '/_config/httpd/max_http_request_size')}` +
           ` && curl -X PUT ${couchConfig.withPath(couchConfig.dbName)}`
       },
-      'reset-test-databases': {
-        stderr: false,
-        cmd: ['medic-test', 'medic-test-audit', 'medic-test-user-admin-meta', 'medic-test-sentinel', 'medic-test-users-meta']
-          .map(
-            name => `curl -X DELETE ${couchConfig.withPath(name)}`
-          )
-          .join(' && '),
+      'setup-test-database': {
+        cmd: [
+          `docker run -d -p 4984:5984 -p 4986:5986 --name e2e-couchdb --mount type=tmpfs,destination=/opt/couchdb/data apache/couchdb:latest`,
```

Comment:
Hey, I'm trying to understand the use of mount here. It's only to be used as a temporary volume right? Running the container and removing it would do that. Also, it seems that tmpfs is limited to linux only. 

> Limitations of tmpfs mounts
Unlike volumes and bind mounts, you can’t share tmpfs mounts between containers.
This functionality is only available if you’re running Docker on Linux.

This could be

`docker run -d -p 4984:5984 -p 4986:5986 --rm --name e2e-couchdb apache/couchdb:latest` and then the tear down could remove the`docker rm -v e2e-couchdb`

Reply:
Sorry I didn't realise tmpfs only works on linux, that's definitely not cool! Your changes listed above all sound good: if you want to raise a PR for that it would be appreciated!

Once I get time and I can get e2e tests to reliably pass locally I'll bench the difference for tmpfs on linux and if it's reasonable I'll add it back safely (ie do it on linux but not on other platforms)